includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\Plugin_Readme_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+/**
+ * Check the plugins readme.txt file and contents.
+ *
+ * @since n.e.x.t
+ */
+class Plugin_Readme_Check extends Abstract_File_Check {
+
+	/**
+	 * Check the readme.txt file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Check_Result $result The Check Result to amend.
+	 * @param array        $files  Array of plugin files.
+	 */
+	protected function check_files( Check_Result $result, array $files ) {
+		// Find the readme file.
+		$readme = self::filter_files_by_regex( $files, '/readme\.txt/' );
+
+		// If the readme.txt does not exist, add a warning and skip other tests.
+		if ( empty( $readme ) ) {
+			$result->add_message(
+				false,
+				__( 'The plugins readme.txt does not exist.', 'plugin-check' ),
```

Comment:
```suggestion
				__( 'The plugin readme.txt does not exist.', 'plugin-check' ),
```

or alternatively "The plugin's ..."

Reply:
Thanks @felixarntz I've updated this based on your suggestion.